test/cases/regression.2422.css

```diff
@@ -0,0 +1,13 @@
+.foobar {
+  border-radius: 4px;
+  columns: 5;
+  foo: 1;
+  foo: 2;
+  foo: 3;
+}
+@media (min-width: 600px) {
+  .foobar {
+    columns: 1;
+    font-weight: 700;
+  }
+}
```

Comment:
Add newline here, thx

Reply:
Done - although the `cases` directory as a whole is quite inconsistent about file-ending newlines. Maybe there should be another PR to bring them all in line?